SofaKernel/modules/SofaHelper/src/sofa/helper/system/FileRepository.cpp

```diff
@@ -91,6 +91,7 @@ FileRepository PluginRepository(
     {
         Utils::getSofaPathTo("plugins"),
         Utils::getSofaPathTo("lib"),
+        Utils::getSofaPathTo("external_directories"),
```

Comment:
Would it make sense to add it to `WIN32` as well?

Reply:
I'm not familiar with the build folder hierarchy for Windows, but if the external plugins are handled the same way, I guess it would make sense to extend the modification

Reply:
@fredroy @guparan @hugtalbot Should the changes be applied on Windows as well?